src/Validation/Validation.php

```diff
@@ -324,20 +336,18 @@ public static function compareWith($check, $field, $context)
      *
      * @param mixed $check The value to find in $field.
      * @param string $field The field to check $check against. This field must be present in $context.
-     * @param bool $result The expected result of field comparison.
+     * @param string $operator Comparison operator.
```

Comment:
I think we need to keep bool for BC instead 3.x? `string|bool ... Deprecated: bool is deprecated.` etc.
Also please add a test for the old bool to still work as expected until we can safely remove in 4.0 then.

Reply:
This method has been added in 3.6. Since 3.6 hasn't been released yet, we shouldn't worry about BC, am I right?